scipy/special/_add_newdocs.py

```diff
@@ -1274,16 +1274,16 @@ def add_newdoc(name, doc):
     Returns
     -------
     scalar or ndarray
-        Value of the incomplete beta function
+        Value of the regularized incomplete beta function
 
     See Also
     --------
     beta : beta function
-    betaincinv : inverse of the incomplete beta function
+    betaincinv : inverse of the regularized incomplete beta function
 
     Notes
     -----
-    The incomplete beta function is also sometimes defined
+    The regularized incomplete beta function is also sometimes defined
```

Comment:
The modified sentence doesn't really work anymore:

* The *regularized* incomplete beta function would never be defined without the `gamma` terms.
* If we're now explicitly calling it the regularized incomplete beta function, there is no need to refer to the "so-called regularized incomplete beta function".

I think the entire paragraph now needs editing.  How replacing it with something like this?

    The term *regularized* in the name of this function refers to the
    scaling of the function by the gamma function terms shown in the
    formula.  When not qualified as *regularized*, the name *incomplete
    beta function* often refers to the just the integral expression,
    without the gamma terms.  One can use the function `beta` from
    `scipy.special` to get this "nonregularized" incomplete beta
    function by multiplying the result of ``betainc(a, b, x)`` by
    ``beta(a, b)``.